forc-util/src/cli.rs

```diff
@@ -131,11 +110,7 @@ macro_rules! cli_examples {
             Box::leak( [
             $(
             $crate::paste::paste! {
-                if stringify!($command) == "forc" {
-                    format!("  #{}\n  forc {}\n\n", stringify!($($description)*), $args )
-                } else {
-                    format!("  #{}\n  forc {} {}\n\n", stringify!($($description)*), stringify!($command), $args )
-                }
+                format!(" # {}\n {}\n\n", stringify!($($description)*), $command)
```

Comment:
One more little nit, we should match the indentation of the usage section:

```suggestion
                format!("    # {}\n    {}\n\n", stringify!($($description)*), $command)
```